rxdiffadapter/src/main/java/com/revolut/rxdiffadapter/RxDiffAdapter.kt

```diff
@@ -47,7 +44,7 @@ open class RxDiffAdapter @Deprecated("Replace with constructor without delegates
     val async: Boolean = false,
     private val autoScrollToTop: Boolean = false,
```

Comment:
Is there a reason to keep autoScrollToTop and detectMoves as properties? They can be declared as simple params instead